start-mycroft.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```

Comment:
BLOCKED by https://github.com/MycroftAI/mycroft-core/pull/2686/files

(pending that, then a rebase or merge to update - then we can review this diff a little more cleanly)